internal/commands/relationship.go

```diff
@@ -264,7 +264,7 @@ func writeRelationshipCmdFunc(operation v1.RelationshipUpdate_Operation) func(cm
 		if operation != v1.RelationshipUpdate_OPERATION_DELETE {
 			caveatString := cobrautil.MustGetString(cmd, "caveat")
 			if caveatString != "" {
-				parts := strings.Split(caveatString, ":")
+				parts := strings.SplitN(caveatString, ":",2)
```

Comment:
so what this solves is the scenario where your caveat value also includes a colon? Mind sharing your example?